Extensions/DialogueTree/dialoguetools.js

```diff
@@ -686,7 +708,9 @@ gdjs.dialogueTree.loadState = function(inputVariable) {
       gdjs.evtTools.network.variableStructureToJSON(inputVariable)
     );
     gdjs.dialogueTree.runner.visited = loadedState.visited;
-    gdjs.dialogueTree.runner.variables.data = loadedState.variables;
+    Object.entries(gdjs.dialogueTree.runner.variables).forEach(function(pair) {
```

Comment:
This will probably not work on Android 4.4: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries

Android 4.4. has Chromium 30. Wdyt?

Reply:
Oh I didn't know. I can change this into a for loop or something

Reply:
updated now